src/Config/SettingsTab.ts

```diff
@@ -54,6 +54,20 @@ export class SettingsTab extends PluginSettingTab {
             text: 'Changing any settings requires a restart of obsidian.',
         });
 
+        new Setting(containerEl)
+            .setName('Task backlink behavior')
+            .setDesc("Decide what happens when clicking on a task's backlink.")
+            .addDropdown((drop: DropdownComponent) => {
+                const settings = getSettings();
+                drop.addOption('taskHeading', "Go to task's heading");
+                drop.addOption('taskLine', "Go to task's line");
+                drop.setValue(settings.backlinkBehavior);
+                drop.onChange(async (value) => {
+                    updateSettings({ backlinkBehavior: value as GotoTaskType });
+                    await this.plugin.saveSettings();
```

Comment:
I can't imagine anyone preferring to jump to the section once it is possible to jump to the line instead.

I would have been inclined to call this a bug-fix, without a preference - for a first release.

And in the incredibly unlikely event that multiple people complain, then to add a preference later.

Reply:
Even better :)

Reply:
Done. The setting is gone.